File: scrudbeans-integration-tests/src/test/java/mypackage/test/SwaggerStaticExporterIT.java
package mypackage.test;


import lombok.extern.slf4j.Slf4j;
import mypackage.ScrudBeansSampleApplication;
import org.junit.jupiter.api.extension.ExtendWith;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit.jupiter.SpringExtension;

/**
 * {@inheritDoc}
 */
@Slf4j
@ExtendWith(SpringExtension.class)
@SpringBootTest(classes = ScrudBeansSampleApplication.class, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
public class SwaggerStaticExporterIT extends com.github.manosbatsis.scrudbeans.test.SwaggerStaticExporterIT {

	private static final Logger LOGGER = LoggerFactory.getLogger(SwaggerStaticExporterIT.class);

}

File: scrudbeans-integration-tests-kotlin/src/test/kotlin/mykotlinckage/test/SwaggerStaticExporterIT.kt
package mykotlinpackage.test

import org.junit.jupiter.api.extension.ExtendWith
import org.slf4j.LoggerFactory
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT
import org.springframework.test.context.junit.jupiter.SpringExtension

/**
 * {@inheritDoc}
@ExtendWith(SpringExtension::class)
@SpringBootTest(classes = [mykotlinpackage.ScrudBeansSampleApplication::class], webEnvironment = RANDOM_PORT)
object SwaggerStaticExporterIT : com.github.manosbatsis.scrudbeans.test.SwaggerStaticExporterIT() {
    private val LOGGER = LoggerFactory.getLogger(SwaggerStaticExporterIT::class.java)
}
 */